src/control/server/mgmt_pool.go

```diff
@@ -26,7 +26,8 @@ import (
 const (
 	// DefaultPoolScmRatio defines the default SCM:NVMe ratio for
 	// requests that do not specify one.
-	DefaultPoolScmRatio = 0.06
+	DefaultPoolScmRatio  = 0.06
+	DefaultPoolNvmeRatio = 0.94
```

Comment:
Hmm. I understand that this was probably used for the sake of expediency in getting this work integrated, but it seems like it would be better to just make the code future proof now. As we discussed, I think a reasonable default here for the Bdev tiers would be `(1 - DefaultPoolScmRatio) / len(req.GetTierratio()))`, i.e. allocate the remaining non-SCM portion of the storage evenly among the Bdev tiers.

Reply:
I don't think this is reasonable, i.e. the tiers will differ in size. Optane capacity will be likely much lower than e.g. a QLC NAND.

Reply:
Ok, fair enough. My point was more that the code here should set some kind of default values and not assume a fixed number of tiers. I am fine with adding support for improved tier-aware defaults in follow-on work.